standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/repl/ReplScope.java

```diff
@@ -93,26 +91,13 @@ public boolean tableIncludedInReplScope(final String tableName) {
     return (inTableIncludedList(tableName) && !inTableExcludedList(tableName));
   }
 
-  private List<Pattern> compilePatterns(List<String> patterns) {
-    if (patterns == null || patterns.isEmpty()) {
+  private Pattern compilePatterns(String pattern) {
```

Comment:
Name can be compilePattern as it is single pattern.

Reply:
done